src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,18 @@
-export const TodoList = () => {};
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC <Props> = ({ todos }) => (
+  <section className="TodoList">
```

Comment:
Consider using a more descriptive class name for the section element. For example, `todo-list-container`. 📝